home/lncm/compose/tor/docker-compose.yml

```diff
@@ -1,9 +1,11 @@
 version: '3.7'
 services:
   bitcoind:
-    image: lncm/bitcoind:0.17.1
+    image: lncm/bitcoind:0.18.0-linux-arm
```

Comment:
Docker manifest should be added to avoid hardcoding architecture, see: https://github.com/lncm/docker-bitcoind/blob/0.18-test/.travis/docker-manifest.sh

Reply:
thats true but 0.18.0 using your image doesn't exist

Reply:
Only arm image was missing, if you built it, you can just create manifest on top of it.

Reply:
can you build an arm version? Don't have a free ARM device right now to do it